playwright/pages/shared/TypeBuilderPage.ts

```diff
@@ -129,6 +130,9 @@ export class TypeBuilderPage extends BuilderPage {
       name: "Remove slice",
       exact: true,
     });
+    this.uidEditorDialog = page
```

Comment:
⚠️ #issue

We are using the POM pattern in our playwright tests. Here you didn't follow how it's done for other Dialog and everything is directly in `TypeBuilderPage`. It should be separated to respect our current pattern.

Reply:
done